client/src/components/App/Content/InvestigationForm/useInvestigationForm.ts

```diff
@@ -158,6 +172,10 @@ const useInvestigationForm = (): useInvestigationFormOutcome => {
         });
     };
 
+    useEffect(() => {
+        fetchIdentificationTypes();
+    }, []);
```

Comment:
maybe add reference to if the redux store? or use Thunk? just check if the current store state is empty and then if it is - fetch it

Reply:
added check if the current store state is empty and then just if only empty then fetch it